tests/nested/manual/core20-new-snapd-does-not-break-old-initrd/task.yaml

```diff
@@ -1,120 +0,0 @@
-summary: verify that new snapd's do not break old snap-bootstrap/kernel initrds
-
-details: |
-  The snapd secure boot implementation is complex, and some of the elements that live in the
-  snapd.git repository end up as a part of the signed kernel package's initrd
-  file. This inadvertently allows a skew to occur, where a future snapd is booted with a past
-  initrd.
-
-  The test exercises two scenarios:
-  
-  1. We start with stable kernel + stable snapd -> refresh to new snapd
```

Comment:
shouldn't we have kept this scenario or is it tested elsewhere?

Reply:
I could try to keep that one. Though for FDE we will have a new test for factory reset that covers it: https://github.com/canonical/snapd/pull/14471

Reply:
I have created https://github.com/canonical/snapd/pull/14957 to see.